### PR TITLE
Use __version__ convention

### PR DESCRIPTION
**Pull request by h-m-f-t**
This PR follows what @IanLee1521 did for pshtt (https://github.com/dhs-ncats/pshtt/pull/107) and aligns versioning following [PEP-396](https://www.python.org/dev/peps/pep-0396/).

### CODE REVIEW

File: trustymail/cli.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
 Notes:
    If no scan type options are specified, all are run against a given domain/input.
 """
+from . import __version__
```
Comment (IanLee1521) [inline trustymail/cli.py]: I would tend to prefer the more explicit: `from trustymail import __version__`

File: trustymail/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+__version__ = '0.2'
```
Comment (IanLee1521) [inline trustymail/__init__.py]: I sense you might be trying to do an actual release; if not I would suggest adding the `-dev` suffix to this.

Additionally, I personally always go with the semantic versioning `X.Y.Z` scheme, so I would change this to `0.2.0`

Comment (jsf9k) [reply trustymail/__init__.py]: I agree with @IanLee1521 on both points.

Comment (jsf9k) [reply trustymail/__init__.py]: I agree with @IanLee1521 on both points, especially for first one.

File: trustymail/cli.py (hunk 1 above)
Comment (jsf9k) [reply trustymail/cli.py]: Agreed

File: setup.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
 setup(
 
         name='trustymail',
-        version='0.0.1',
+        version=__version__,
```
Comment (IanLee1521) [inline setup.py]: It looks like `setup.py` is missing the `from trustymail import __version__` call.

Comment (jsf9k) [reply setup.py]: @IanLee1521 Are you saying there is one bug left to iron out?  I think I've lost the thread here...it looks like all the requested changes have been made.

Comment (IanLee1521) [reply setup.py]: correct, currently there is no import in that file to get that value... (Only in the other PR).

**Pull request merged**